tests/e2e/query_modules/workloads.yaml

```diff
@@ -26,3 +26,9 @@ workloads:
     proc: "query_modules/"
     args: ["query_modules/mgps_test.py"]
     <<: *in_memory_cluster
+
+  - name: "Schema query module test"
+    binary: "tests/e2e/pytest_runner.sh"
+    proc: "query_modules/"
+    args: ["query_modules/schema_test.py"]
+    <<: *in_memory_cluster
```

Comment:
We can't test schema.so as it was flaky. It needs to pass at least 10 times and then we can merge it. Check PR for more info: https://github.com/memgraph/memgraph/pull/1675

Reply:
Thanks for the heads-up! I’ve since removed this workload